### PR TITLE
Added TSG to VMSS task for Permission Denied

### PR DESCRIPTION
**Pull request by t-dedah**
This issue was faced by one of the customer and this fix solved it. [Link to DTS](https://dev.azure.com/mseng/AzureDevOps/_queries/edit/1811707/?triage=true)

### CODE REVIEW

**Comment by PRMerger6** [comment]
@t-dedah : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by pulkitaggarwl — APPROVED** [review]

File: docs/pipelines/tasks/deploy/azure-vmss-deployment.md [hunk 1]
```diff
@@ -46,6 +46,22 @@ Use this task to deploy a virtual machine scale set image.
 
 </table>
 
+
+## Troubleshooting
+
+### Error: Permission denied
+
+This issue happens when we try to run a custom script but the script isn't executable.
```
Comment (bishal-pdMSFT) [inline docs/pipelines/tasks/deploy/azure-vmss-deployment.md:54]: Can you give a more detailed error message. Just `Permission denied` is very generic.

File: docs/pipelines/tasks/deploy/azure-vmss-deployment.md [hunk 2]
```diff
@@ -46,6 +46,22 @@ Use this task to deploy a virtual machine scale set image.
 
 </table>
 
+
+## Troubleshooting
+
+### Error: Permission denied
+
+This issue happens when we try to run a custom script but the script isn't executable. 
+First make sure the customScript variable doesnt have './' or anything else before the script name(test.sh)
```
Comment (bishal-pdMSFT) [inline docs/pipelines/tasks/deploy/azure-vmss-deployment.md:55]: not `variable`. it is an `input`

**Review by bishal-pdMSFT — APPROVED** [review]

**Comment by PRMerger8** [comment]
@t-dedah : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger17** [comment]
@steved0x : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by steved0x** [comment]
#sign-off

**Comment by PRMerger17** [comment]
@steved0x: I'm sorry - only the author of this article, @RoopeshNair, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Review by steved0x — APPROVED** [review]

**Comment by PRMerger18** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**